core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -92,6 +92,14 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
     val parameterTaskSortDesc = UIUtils.stripXSS(request.getParameter("task.desc"))
     val parameterTaskPageSize = UIUtils.stripXSS(request.getParameter("task.pageSize"))
```

Comment:
we need to clean this up as its not really used anymore, can you file a jira to change the timeline to use rest api or the data from the other tables..

Reply:
Done, https://issues.apache.org/jira/browse/SPARK-26130.